Formula/druid.rb

```diff
@@ -14,22 +14,29 @@ def install
     cd libexec do
       system "bin/init"
     end
+
+    Pathname.glob("#{libexec}/bin/*.sh") do |file|
```

Comment:
There's a more concise way to do with with `#install_symlink` https://github.com/Homebrew/homebrew-core/blob/master/Formula/coreutils.rb#L92-L99